Source/Core/InputCommon/ControllerEmu/ControlGroup/ControlGroup.cpp

```diff
@@ -28,6 +28,17 @@ ControlGroup::ControlGroup(std::string name_, std::string ui_name_, const GroupT
 {
 }
 
+void ControlGroup::AddVirtualNotchSetting(SettingValue<double>* value, double max_virtual_notch_deg)
+{
+  AddSetting(value,
+             {_trans("Virtual Notches"),
+              // i18n: The degrees symbol.
+              _trans("°"),
+              // i18n: Virtual Notch snap for circular controller gates.
+              _trans("Virtual Notch snap for circular controller gates.")},
```

Comment:
I don't really like the "for circular controller gates" bit. This feature applies to any shape of stick.

Reply:
Changed it to:
> Snap the thumbstick position to the nearest octagonal axis.

Finding it a bit tricky to properly define this in such a short space but I think this somewhat does the job

Reply:
Good enough for me.